graylog2-bootstrap/src/main/java/org/graylog2/bootstrap/ServerBootstrap.java

```diff
@@ -74,8 +75,10 @@ public boolean isNoPidFile() {
 
     @Override
     protected void startCommand() {
-        LOG.info("Graylog {} {} starting up (JRE: {}, installed from: {})", commandName, version,
-                Tools.getSystemInformation(), configuration.getInstallationSource());
+        final OS os = OS.getOs();
+
+        LOG.info("Graylog {} {} starting up (JRE: {}, installed from: {}, OS: {})", commandName, version,
```

Comment:
Sure, we can do that. I wanted to save some log lines. :stuck_out_tongue_winking_eye: 
Something like this?

```
2015-11-11 17:35:14,679 INFO : org.graylog2.bootstrap.ServerBootstrap - Graylog server 1.3.0-SNAPSHOT (91bbaeb) starting up
2015-11-11 17:35:14,679 INFO : org.graylog2.bootstrap.ServerBootstrap - Java Version: Oracle Corporation 1.8.0_66 on Linux 3.19.0-32-generic
2015-11-11 17:35:14,679 INFO : org.graylog2.bootstrap.ServerBootstrap - Installation Source: deb
2015-11-11 17:35:14,679 INFO : org.graylog2.bootstrap.ServerBootstrap - OS: Ubuntu 14.04.3 LTS (trusty)
2015-11-11 17:35:14,679 INFO : org.graylog2.bootstrap.ServerBootstrap - Arch: amd64
```